java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/MoveClassUI.java

```diff
@@ -305,7 +305,10 @@ public RefactoringUI create(CompilationInfo info, TreePathHandle[] handles, File
                 Collection<TreePathHandle> tphs = new ArrayList<TreePathHandle>();
                 SourcePositions sourcePositions = info.getTrees().getSourcePositions();
                 for (Element ele : e.getEnclosedElements()) {
-                    Tree leaf = info.getTrees().getPath(ele).getLeaf();
+                    TreePath path = info.getTrees().getPath(ele);
+                    if(path == null)
```

Comment:
Please use curly braces (`{}`) and reformat your changes (select lines > format) (missing white space after `if`)

Reply:
comment addressed